color.go

```diff
@@ -150,6 +156,30 @@ func New(value ...Attribute) *Color {
 	return c
 }
 
+// RGB returns a newly created foreground 24-bit RGB color.
+func RGB(r, g, b int) *Color {
+	return New(foreground, 2, Attribute(r), Attribute(g), Attribute(b))
+}
+
+// BgRGB returns a newly created 24-bit background RGB color.
+func BgRGB(r, g, b int) *Color {
+	return New(background, 2, Attribute(r), Attribute(g), Attribute(b))
+}
+
+// AddRGB is used to chain foreground RGB SGR parameters. Use as many as parameters to combine
+// and create custom color objects. Example: .Add(34, 0, 12).Add(255, 128, 0).
+func (c *Color) AddRGB(r, g, b int) *Color {
+	c.params = append(c.params, foreground, 2, Attribute(r), Attribute(g), Attribute(b))
+	return c
+}
+
+// AddRGB is used to chain background RGB SGR parameters. Use as many as parameters to combine
+// and create custom color objects. Example: .Add(34, 0, 12).Add(255, 128, 0).
```

Comment:
Reading through the example: `Example: .Add(34, 0, 12).Add(255, 128, 0)`

Is it worth including an example in the README with the ability to chain? I don't think we have that now?